clueless/settings.py

```diff
@@ -118,3 +117,5 @@
 # https://docs.djangoproject.com/en/1.10/howto/static-files/
 
 STATIC_URL = '/static/'
+
+from .settings_secret import *
```

Comment:
OOhhhhhhh- hover over that symbol.  It means that there isn't a newline character at the end of the file.  I'll add it and commit.